src/components/pages/order/Navbar/Rightside.jsx

```diff
@@ -1,10 +1,41 @@
+import { useState } from "react"
+import { toast } from "react-toastify"
 import styled from "styled-components"
 import Profile from "./Profile"
+import ToggleButton from "../../../reusable-ui/ToggleButton"
+import Toast from "./Toast"
 
 export default function Rightside() {
+  const [checked, setChecked] = useState(false)
```

Comment:
```suggestion
  const [isChecked, setIsChecked] = useState(false)
```

c'est un boolean, on va privilégier des noms de variables commençant par "is" qqch ou "has" qqch ou encore "can" qqch comme si on posait une question.

Reply:
Autre chose: même isChecked c'est pas terrible terrible car là, tu raisonnes d'un point vu de dev, raisonnes d'un point de vue "métier" ou "utilisateur" pour trouver un nom de variable plus "parlant" tout en respectant les bonnes pratiques de nommage en JS